src/Encrypt.java

```diff
@@ -0,0 +1,117 @@
+package src;
+// This is a singleton class so that only one instance of this class is ever created and is in use
+
+import java.nio.charset.Charset;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Random;
```

Comment:
Because this is a security application be sure to instead use a secure random number generator anywhere you are using random numbers.  You can do by using this package instead:  import java.security.SecureRandom;